evaluation/DPVerification.py

```diff
@@ -5,12 +5,15 @@
 # i.e. passing (epsilon, delta) - DP condition
 # If the definition is not passed, there is a bug or it is a by-design bug in case of passing actual aggregates
 
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
 import pandas as pd
 import numpy as np
 import math
 import matplotlib.pyplot as plt
-import Aggregation as agg
-import os
+import evaluation.Aggregation as agg
+import evaluation.Exploration as exp
```

Comment:
same naming convention comment from before for exploration.py

Reply:
Renamed to exploration.py instead of Exploration.py